frontend/test/lit/test_mcmc.py

```diff
@@ -34,7 +34,7 @@
     )
 )
 def circuit():
-    return qml.state()
+    return qml.expval(qml.X(0))
 
 
 # CHECK: 'mcmc': True, 'num_burnin': 200, 'kernel_name': 'NonZeroRandom'
```

Comment:
The test doesn't even need shots, weird 😅 Either way, thanks for the fix!

Reply:
For `mcmc=True` on the device, the number of shots needs to exceed `num_burnin`. Removing shots raises an error on device initialization.

Reply:
Ahh, good to know :D